update-docker-repo-info.py

```diff
@@ -92,6 +104,34 @@ def get_os_release(image_name):
     return res.stdout.splitlines()
 
 
+def get_python_version(docker_info: str) -> str:
```

Comment:
not related to this pr, but I noticed this env var r'PYTHON_VERSION doesnt exist for the ubi image. Maybe we should add it in the Dockerfile manually? Lets discuss

Reply:
https://github.com/demisto/dockerfiles/pull/19210

Reply:
merged